packages/webawesome/src/styles/utilities/text.css

```diff
@@ -126,4 +118,26 @@
       }
     }
   }
+  /* #endregion */
+
+  /* #region Form Controls */
```

Comment:
```suggestion
  /* #region Form Controls ~~~~~~~ */
```

You were missing them squigglies here. Need 'em? Also, what do they do - just easy find in file business?

Reply:
I use 'em to align the right edge of `#region` headings so that they have a clean edge when regions are collapsed. So, I'll add just enough squigglies to each line to match the length of the longest one, like so:
<img width="237" height="129" alt="Screenshot 2025-10-15 at 1 15 38 PM" src="https://github.com/user-attachments/assets/61b895ae-c3e9-45e3-8d6c-7730642ddeb5" />

The same convention is at play in `native.css`.

We could go no-squiggly if it's more burdensome/confusing than its worth, though. Thoughts?

Reply:
> _We could go no-squiggly if it's more burdensome/confusing than its worth, though. Thoughts?_

Now that I've learned of the squigglies, no-squiggly is a no-go for me.  I'll sync up the other regions in other CSS files if I see any out of whack in my travels!